src/main/java/views/AcesUp/AcesUp.flt.html

```diff
@@ -62,6 +67,12 @@
     });
 
     var game;
+    var isSpanish = 0;
+    function isSpain(input){
+        isSpanish = input;
+    }
+
```

Comment:
Is it okay if I make some adjustment later? Overall, the logic is similar to what we discussed in group meeting.